### PR TITLE
doc/userguide: document logging changes from 6 to 7

### PR DESCRIPTION
**Pull request by victorjulien**
Minor other logging related improvements like clarifying language and improving formatting for pdf output.

### CODE REVIEW

File: doc/userguide/configuration/suricata-yaml.rst [hunk 1]
```diff
@@ -1816,23 +1816,23 @@ generated alerts and events.
...
 compiled with the ``--enable-debug`` configure option.
 
 The first option within the logging configuration is the
-default-log-level. This option determines the severity/importance
+``default-log-level``. This option determines the severity/importance
 level of information that will be displayed. Messages of lower levels
 than the one set here, will not be shown. The default setting is
-Info. This means that error, warning and info will be shown and the
-other levels won't be.
+``Info``. This means that ``error``, ``warning`` and ``info`` will be shown
```
Comment (jufajardini) [inline doc/userguide/configuration/suricata-yaml.rst:1834]: What about `notice`?

File: doc/userguide/configuration/suricata-yaml.rst [hunk 2]
```diff
@@ -1974,18 +1974,18 @@ computers etc.)
...
-        facility: local5                          #In this option you can set a syslog facility.
-        format: "[%i] <%d> -- "                   #The option to set your own format.
-        #level: notice                            #Use a different level than the default.
+    - console:                          # Output to screen (stdout/stderr).
+        enabled: yes                    # This option is enabled.
+        #level: notice                  # Use a different level than the default.
+    - file:                             # Output stored in a file.
+        enabled: no                     # This option is not enabled.
+        filename: /var/log/suricata.log # Filename and location on disc.
+        level: info                     # Use a different level than the default.
```
Comment (jufajardini) [inline doc/userguide/configuration/suricata-yaml.rst:1983]: But isn't `info` the default?

File: doc/userguide/configuration/suricata-yaml.rst [hunk 3]
```diff
@@ -1974,18 +1974,18 @@ computers etc.)
...
-        filename: /var/log/suricata.log           #Filename and location on disc.
-        level: info                               #Use a different level than the default.
-    - syslog:                                     #This is a program to direct log-output to several directions.
-        enabled: no                               #The use of this program is not enabled.
-        facility: local5                          #In this option you can set a syslog facility.
-        format: "[%i] <%d> -- "                   #The option to set your own format.
-        #level: notice                            #Use a different level than the default.
+    - console:                          # Output to screen (stdout/stderr).
+        enabled: yes                    # This option is enabled.
+        #level: notice                  # Use a different level than the default.
```
Comment (jufajardini) [inline doc/userguide/configuration/suricata-yaml.rst:1979]: considering this is commented out, doesn't this actually imply we'll use the default?

File: doc/userguide/configuration/suricata-yaml.rst (hunk 1 above)
Comment (victorjulien) [reply doc/userguide/configuration/suricata-yaml.rst:1834]: hah, thats what I get for doing style fixups :-P

File: doc/userguide/configuration/suricata-yaml.rst (hunk 2 above)
Comment (victorjulien) [reply doc/userguide/configuration/suricata-yaml.rst:1983]: no, notice is

File: doc/userguide/configuration/suricata-yaml.rst (hunk 3 above)
Comment (victorjulien) [reply doc/userguide/configuration/suricata-yaml.rst:1979]: Yeah, but what do you want to do? Add "only if not commented out" everywhere? Seems kind of unnecessary.

**Comment by victorjulien** [comment]
replaced by #11566

**Pull request closed without merging**